README.md

```diff
@@ -18,7 +22,12 @@ The library creates and manages per-zoom-level clusters for large amounts of mar
 
 See the [history section](#history) and [migration section](#migration) for how this library relates to [@google/markerclusterer][@google/markerclusterer] and [@googlemaps/markerclustererplus][@googlemaps/markerclustererplus].
 
-## Install
+## Requirements
+
+- A modern frontend framework like Angular, React, or Vue.js
```

Comment:
I think we can just remove this from requirements. It can work together with those (debatable how well that actually works), but usage in vanilla applications is perfectly fine as well.

```suggestion
```